authors/apps/articles/models.py

```diff
@@ -49,10 +49,13 @@ class Comment(models.Model):
     body = models.TextField(blank=False)
     createdAt = models.DateTimeField(auto_now_add=True)
     updatedAt = models.DateTimeField(auto_now=True)
+    selected_text = models.TextField(null=True)
```

Comment:
I think this implies that I can only highlight one text per article. That assumption would be wrong, I should be able to highlight several pieces of text in an article but these highlights should only be visible to me. I think this will involve creating another table that will have user_id, article_id, highlight_start and highlight_end